src/System.Private.CoreLib/shared/System/Globalization/CompareInfo.Unix.cs

```diff
@@ -852,33 +852,59 @@ internal unsafe int GetHashCodeOfStringCore(ReadOnlySpan<char> source, CompareOp
                 return 0;
             }
 
-            fixed (char* pSource = source)
+            // according to ICU User Guide: 
```

Comment:
What is the license for the ICU User Guide?

In general, quoting from documentation may require updating https://github.com/dotnet/coreclr/blob/master/THIRD-PARTY-NOTICES.TXT or it may be downright problematic if the license for the documentation is not compatible.

Reply:
I have removed the quotes, thanks for pointing this out.